src/main/java/insat/company/platform/domain/Club.java

```diff
@@ -0,0 +1,111 @@
+package insat.company.platform.domain;
+
+
+import javax.persistence.*;
+
+import org.springframework.data.elasticsearch.annotations.Document;
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * A Club.
+ */
+@Entity
+@Table(name = "club")
+@Document(indexName = "club")
+public class Club implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "president")
+    private String president;
```

Comment:
This should be a relationship with a User not a String

Reply:
ok I'll fix it